docs/api_documentation.md

```diff
@@ -651,6 +644,15 @@ e.g.:
 curl -i -H "Accept: application/json" -H "Content-type: application/json" -H "Authorization: Bearer $YOUR_ACCESS_TOKEN" -X DELETE http://localhost:8080/osiam/Users/$ID
 ```
 
+### Get the Authenticated User
```

Comment:
Please add to TOC. Shouldn't it be within the `User management` section? And if you like, please remove dead link to `Get the User of the current accessToken`.